test/core/byoa/test_fileset.py

```diff
@@ -1,20 +1,20 @@
 import os
 import os.path
 import os.path
+import os.path
```

Comment:
```suggestion
import os.path
```

I think once is enough!

Reply:
:)